TestProjects/ShaderFoundry/Assets/CommonAssets/Editor/EditorTests/FeatureTests/FallbackTests.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using UnityEditor.ShaderGraph;
+
+namespace UnityEditor.ShaderFoundry.UnitTests
+{
+    class FallbackTests // : BlockTestRenderer
+    {
+        [Test]
+        public void NoFallback_ShowsDefaultInGeneratedShader()
```

Comment:
nit: I might rename these to be a bit clearer. "NoFallbackSet_DefaultFallbackIsUsed"